modules/kitty/hm.nix

```diff
@@ -32,7 +32,10 @@ in {
         size = config.stylix.fonts.sizes.terminal;
       };
       settings.background_opacity = with config.stylix.opacity; "${builtins.toString terminal}";
-      extraConfig = builtins.readFile theme;
+      extraConfig = ''
+        include stylix.conf
```

Comment:
Could this work without linking another file in the home directory?
```suggestion
        include ${theme}
```

Reply:
Probably, let me give it a shot

Reply:
Yup! Done :)